src/lib.rs

```diff
@@ -23,18 +22,48 @@ pub struct Fernet {
 #[derive(Debug, PartialEq, Eq)]
 pub struct DecryptionError;
 
+pub struct MultiFernet {
+    fernets: Vec<Fernet>,
+}
+
+impl MultiFernet {
+    pub fn new(keys: Vec<String>) -> MultiFernet {
```

Comment:
This should just take a `Vec<Fernet>` like we do in the Python version.

There should also be an `assert`/`panic` that there's at least one element in the vec